### PR TITLE
Translate content from TbA+ritsPkyk and some others

### PR DESCRIPTION
**Pull request by robflop**

### CODE REVIEW

File: src/data/skill-desc.csv [hunk 1]
```diff
@@ -51,9 +51,9 @@
 "レース終盤に前方が詰まると\nお告げの力で道を切り開く","<size=22>When the front is packed during the last leg of the race, the oracle's power will open a path</size>"
 "レース終盤で3番手の時に負けそうにな\nると秘めた闘志に火が付き速度が上がる","<size=22>When you're in 3rd place at the end of the race and about to lose, your speed increases</size>"
 "レースを冷静に運ぶことができると残り\n200mで実力発揮、速度がすごく上がる","<size=22>After proceeding through the race calmly, your speed drastically increases during the last 200m</size>"
-"中盤で2回スキルを使うと\nよりリラックスして持久力が回復する",""
+"中盤で2回スキルを使うと\nよりリラックスして持久力が回復する","<size=22>Restore endurance if you activate two skills in the middle part of the race</size>"
 "最終直線で前の方で追いすがっていると\n不屈の闘志で速度が上がる","<size=22>Increase speed in the last straight when chasing someone close to the lead</size>"
-"スキルで持久力を回復すると\nレース後半で前のほうにいる時に\nレッツラゴー！と速度を上げる",""
+"スキルで持久力を回復すると\nレース後半で前のほうにいる時に\nレッツラゴー！と速度を上げる","<size=22>In the second half of the race, increase speed when you recover endurance with your skills while being in front</size>"
```
Comment (FabulousCupcake) [inline src/data/skill-desc.csv:56]:
```suggestion
"スキルで持久力を回復すると\nレース後半で前のほうにいる時に\nレッツラゴー！と速度を上げる","<size=22>In second half of the race, increase speed when you recover endurance with your skills while being in front</size>"
```

Comment (FabulousCupcake) [reply src/data/skill-desc.csv:56]: This is so long… I think only this can be removed without making it unclear

File: src/data/skill-desc.csv [hunk 2]
```diff
@@ -383,13 +383,13 @@
 "すぐ後ろにウマ娘が長くいると\n速度がわずかに上がる","<size=22>Slightly increase speed when a girl has been following you for a while</size>"
 "レース中盤に競り合うと\n速度が上がる＜ダート＞","<size=22>Dirt・Increase speed when competing for position during the Middle Leg</size>"
 "レース中盤に競り合うと\n速度がわずかに上がる＜ダート＞","<size=22>Dirt・Slightly increase speed when competing for position during the Middle Leg</size>"
-"レースへの渇望により\n能力が少し発揮しやすくなる",""
-"かつての走りとは違うがあの頃と\n変わらない想いが能力をすごく上げる",""
+"レースへの渇望により\n能力が少し発揮しやすくなる","<size=22>Use your craving to race to become able to demonstrate your power a little better</size>"
```
Comment (FabulousCupcake) [inline src/data/skill-desc.csv:386]:
```suggestion
"レースへの渇望により\n能力が少し発揮しやすくなる","<size=22>Use your craving to race to be able to demonstrate your power a little better</size>"
```

Comment (FabulousCupcake) [reply src/data/skill-desc.csv:386]: Alternatively
```suggestion
"レースへの渇望により\n能力が少し発揮しやすくなる","<size=22>Use your craving to race to demonstrate your power a little better</size>"
```

File: src/data/skill-desc.csv [hunk 3]
```diff
@@ -383,13 +383,13 @@
...
+"かつての走りとは違うがあの頃と\n変わらない想いが能力をすごく上げる","<size=22>Remembering the slightly different yet unchanging running of the past, raise your ability greatly</size>"
```
Comment (FabulousCupcake) [inline src/data/skill-desc.csv:387]:
```suggestion
"かつての走りとは違うがあの頃と\n変わらない想いが能力をすごく上げる","<size=22>Remembering the slightly different yet unchanging past runs, raise your ability greatly</size>"
```

Comment (FabulousCupcake) [reply src/data/skill-desc.csv:387]: Alternatively
```suggestion
"かつての走りとは違うがあの頃と\n変わらない想いが能力をすごく上げる","<size=22>Remembering the slightly different, yet unchanging runs in the past gives you raise your ability greatly</size>"
```

File: src/data/skill-desc.csv [hunk 4]
```diff
@@ -412,7 +412,7 @@
 "レース後半のコーナーで控えていると\n持久力がわずかに回復する","<size=22>When being held back on corners of the later half of the race, your endurance recovers slightly</size>"
 "最終直線で競り合いに負けない\n気持ちが湧けば、速度がわずかに上がる","<size=22>When competing for the lead during the last straight, your speed increases slightly</size>"
 "レース中間付近で好位置にいる時に\n追い抜くとちょっと進出を開始する","<size=22>Gain slight momentum while in a good position when passing someone in the Middle Leg</size>"
-"レース後半に中団から仕掛け始める時に\n加速力をちょっとずつ上げる",""
+"レース後半に中団から仕掛け始める時に\n加速力をちょっとずつ上げる","<size=22>In the second half of the race, increase acceleration very slightly if you start overtaking from the middle of the pack</size>"
```
Comment (FabulousCupcake) [inline src/data/skill-desc.csv:415]:
```suggestion
"レース後半に中団から仕掛け始める時に\n加速力をちょっとずつ上げる","<size=22>In second half of the race, increase acceleration very slightly if you start overtaking from the middle of the pack</size>"
```

File: src/data/skill-desc.csv [hunk 5]
```diff
@@ -421,9 +421,9 @@
 "レース終盤に前方が詰まると\nお告げ次第で道をわずかに切り開く","<size=22>Slightly easier to find an opening if blocked at the front in Final Leg</size>"
 "レース終盤で3番手の時に負けそうに\nなると速度がわずかに上がる","<size=22>When you're in 3rd place at the end of the race and about to lose, your speed slightly increases</size>"
 "レースを冷静に運ぶことができると残り\n200mで実力発揮、速度が少し上がる","<size=22>After proceeding through the race calmly, your speed increases a little during the last 200m</size>"
-"中盤で2回スキルを使うと\n持久力がわずかに回復する",""
+"中盤で2回スキルを使うと\n持久力がわずかに回復する","Restore endurance slightly if you activate two skills in the middle part of the race"
 "最終直線で前の方で追いすがっていると\n速度がわずかに上がる","<size=22>Slightly increase speed in the last straight when chasing someone close to the lead</size>"
-"スキルで持久力を回復すると\nレース後半で前のほうにいる時に\nわずかに速度を上げる",""
+"スキルで持久力を回復すると\nレース後半で前のほうにいる時に\nわずかに速度を上げる","<size=22>In the second half of the race, increase speed slightly when you recover endurance with your skills while being in front</size>"
```
Comment (FabulousCupcake) [inline src/data/skill-desc.csv:426]:
```suggestion
"スキルで持久力を回復すると\nレース後半で前のほうにいる時に\nわずかに速度を上げる","<size=22>In second half of the race, increase speed slightly when you recover endurance with your skills while being in front</size>"
```

File: src/data/support-title.csv [hunk 6]
```diff
@@ -69,9 +69,9 @@
 "[おもい、ねがう]","[A Feeling, a Wish]"
 "[Head-on fight！]","[Head-on fight!]"
 "[Two Pieces]","[Two Pieces]"
-"[ツメたいヒトリジメ？]",""
-"[全力！タントラムッッ！]",""
-"[トライフル☆バケーション]",""
+"[ツメたいヒトリジメ？]","[Chilly Hogging?]"
+"[全力！タントラムッッ！]","[Full Power! Tantrum!]"
+"[トライフル☆バケーション]","[Trifle ☆ Vacation]"
```
Comment (FabulousCupcake) [inline src/data/support-title.csv:74]: TIL it's a dessert  
https://en.wikipedia.org/wiki/Trifle

File: src/data/trainer-title.csv [hunk 7]
```diff
@@ -173,10 +173,10 @@
...
 "ウイニングチケット専属","Winning Ticket's Exclusive Trainer"
 "ウイニングチケット名手","Masterful Winning Ticket Trainer"
-"ゴールドシチーとの出会い",""
-"ゴールドシチー担当",""
-"ゴールドシチー専属",""
-"ゴールドシチー名手",""
+"ゴールドシチーとの出会い","Memories with Gold City"
+"ゴールドシチー担当","Gold City's Personal Trainer"
+"ゴールドシチー専属","Gold City's Personal Trainer"
+"ゴールドシチー名手","Gold City's Personal Trainer"
```
Comment (FabulousCupcake) [inline src/data/trainer-title.csv:179]: Bad copypaste job

File: src/data/skill-desc.csv (hunk 1 above)
Comment (robflop) [reply src/data/skill-desc.csv:56]: "In race's second half" could make it shorter by changing word order around a bit?
Also "increase speed when recovering endurance with skills", going to submit a suggested change

Comment (robflop) [inline src/data/skill-desc.csv:56]:
```suggestion
"スキルで持久力を回復すると\nレース後半で前のほうにいる時に\nレッツラゴー！と速度を上げる","<size=22>In race's second half, increase speed when recovering endurance with skills while in front</size>"
```

File: src/data/skill-desc.csv (hunk 2 above)
Comment (robflop) [reply src/data/skill-desc.csv]: Going to apply this suggestion but replace power with ability.

File: src/data/support-title.csv (hunk 6 above)
Comment (robflop) [reply src/data/support-title.csv:74]: ikr

File: src/data/trainer-title.csv (hunk 7 above)
Comment (robflop) [reply src/data/trainer-title.csv:179]: Whoops

File: src/data/skill-desc.csv (hunk 5 above)
Comment (robflop) [reply src/data/skill-desc.csv:426]: See below suggestion by me, similar to your other suggestion above

Comment (robflop) [inline src/data/skill-desc.csv:426]:
```suggestion
"スキルで持久力を回復すると\nレース後半で前のほうにいる時に\nわずかに速度を上げる","<size=22>In race's second half, increase speed slightly when recovering endurance with skills while in front</size>""
```

File: src/data/skill-desc.csv (hunk 1 above)
Comment (FabulousCupcake) [reply src/data/skill-desc.csv:56]: `In the race second half` or even `In race second half` sounds even shorter and still correct to me

File: src/data/skill-desc.csv (hunk 5 above)
Comment (FabulousCupcake) [reply src/data/skill-desc.csv:426]: Remove extra quote at the end

File: src/data/skill-desc.csv (hunk 3 above)
Comment (robflop) [reply src/data/skill-desc.csv:387]: I don't think "runs" fits here, as 走り refers to the act of running, and not any one specific run.

File: src/data/skill-desc.csv (hunk 1 above)
Comment (robflop) [reply src/data/skill-desc.csv:56]: Dropping the "the" makes it sound awkward, i'd personally most prefer "In the race's second half" as that sounds most correct and only adds two characters to your first suggestion here, while still being shorter than the original.

Comment (FabulousCupcake) [reply src/data/skill-desc.csv:56]: `In race second half, increase speed when…` actually does not sound awkward to me at all; it actually sounds better than the original longer one if I ignore the length; need a third opinion here

Comment (robflop) [reply src/data/skill-desc.csv:56]: I'm good with waiting for a third opinion

File: src/data/skill-desc.csv (hunk 3 above)
Comment (FabulousCupcake) [reply src/data/skill-desc.csv:387]: `running of the past` sounds very awkward, but I don't have anything better

**Review by FabulousCupcake — DISMISSED** [review]

**Review by FabulousCupcake — APPROVED** [review]

**Pull request merged**